gtsam/navigation/NavState.cpp

```diff
@@ -114,24 +114,23 @@ NavState NavState::inverse() const {
 
 //------------------------------------------------------------------------------
 NavState NavState::Expmap(const Vector9& xi, OptionalJacobian<9, 9> Hxi) {
-  // Get angular velocity w, translational velocity v, and acceleration a
-  Vector3 w = xi.head<3>();
-  Vector3 rho = xi.segment<3>(3);
-  Vector3 nu = xi.tail<3>();
+  // Get angular velocity w and components rho and nu from xi
```

Comment:
$\rho$ should be translation and $\nu$ should be velocity and that should be mentioned in this comment IMHO.

Reply:
Done